src/notifications/notificationRepository.ts

```diff
@@ -1,35 +1,31 @@
 import { injectable } from "inversify";
-import { Database } from "../infra/dataSource";
+import { Database } from "../infra/database";
 import { INotificationRepository } from "./interfaces/INotificationRepository";
-import { Notification } from "./notificationsEntity";
+import { NotificationModel } from "./NotificationModel";
 
 @injectable()
 export class NotificationRepository implements INotificationRepository {
   constructor(private readonly dataSource: Database) {}
 
   async addCode(code: number, user_id: number, type: string) {
-    const repo = this.dataSource.getRepository(Notification);
-    await repo.insert({
+    return NotificationModel.create({
       user_id: user_id,
       code: code,
       type_of_notice: type
     })
   }
 
   async getCode(user_id: number, type_of_notice: string) {
-    const repo = this.dataSource.getRepository(Notification);
-    const getCode = await repo.findOne({where: {user_id: user_id, type_of_notice: type_of_notice}})
+    const getCode = await NotificationModel.findOne({where: {user_id: user_id, type_of_notice: type_of_notice}})
     return getCode;
   }
 
   async checkCode(id: number, code: number) {
-    const repo = this.dataSource.getRepository(Notification);
-    const user = await repo.findOne({where: {user_id: id, code: code}})
-    return user;
+    const user = await NotificationModel.findOne({where: {user_id: id, code: code}})
+    return user!.dataValues;
```

Comment:
why u using dataValues here?
Sequelize models has pretty function .toJSON()
that convert Sequelize model to javascript object